samples/apps/bookinfo/src/build-docker-services.sh

```diff
@@ -0,0 +1,62 @@
+SCRIPTDIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
```

Comment:
comment a bit at the top

Reply:
thanks, resolved.

Reply:
I meant what it does overall "This script builds docker services,.... it's different from 
https://github.com/istio/istio/blob/master/samples/apps/bookinfo/src/build-services.sh
and
https://github.com/istio/pilot/tree/master/docker
because ... "